scripts/tokenomics-emulation/network.ts

```diff
@@ -143,6 +143,23 @@ class Network {
     });
   }
 
+  /**
+   * Calculate the remaining ERA time. 
+   * @returns Era Start time
+   */
+  public async eraTime() {
```

Comment:
@karthikeyan-cere 
Rename the method and comments.

Reply:
Renamed